amundsen_application/static/js/components/Footer/index.tsx

```diff
@@ -30,12 +38,14 @@ export class Footer extends React.Component<FooterProps> {
   };
 
   render() {
-    let content;
+    let content = (<ShimmeringFooterLoader />);
```

Comment:
I actually don't think we want to show the loader if the value doesn't exist, we originally show nothing so that things look less "broken". I forgot that this component doesn't have a loading state, so perhaps it wasn't a good candidate for this yet. 

**But**, I think we could keep it and some future cleanup task would be to allow `Footer` to know when things are loading. So that the UI will show the loader and then either show the date string or no string -- or just not even render the Footer probably. Any opinions?

Reply:
Oh, I see. I thought we would always have a lastIndexed. In that case I agree, we shouldn't show the loader if there won't be any value.

And yes, we probably won't want to load the footer, but that would require some extra tweaking. I am not sure if Knowl would actually asks us to remove the footer overall.

Not sure how to proceed at this moment.

Reply:
We will always have it when things work as expected at Lyft, but this is just an edge case for an issue that can happen in the backend, so we just make the UI account for it as gracefully as possible. 

I'm okay with leaving this code as is though -- perhaps this is one of the cases where its not a terrible thing that something looks broken. Because if it actually is broken (in the backend) this hints that something isn't right.